Gemfile.lock

```diff
@@ -137,6 +137,3 @@ PLATFORMS
 DEPENDENCIES
   github-pages
   html-proofer
-
-BUNDLED WITH
-   1.11.2
```

Comment:
Whoops.  I have a different Ruby version than specified in the Gemfile and so it occasionally makes unrelated changes.  I ran `git commit -a` and pushed without checking the diff, which was bad of me.  I've forced-pushed a new commit that doesn't have this change.

Thanks for catching it!